readme.md

```diff
@@ -36,6 +36,7 @@ Please read the [Contributing Guidelines](./contributing.md) before contributing
   - [Schizophrenia](#schizophrenia)
   - [Borderline Personality Disorder](#borderline-personality-disorder-bpd)
   - [Prosopagnosia](#prosopagnosia)
+  - [Attention deficit hyperactivity disorder](#attention-deficit-hyperactivity-disorder)
```

Comment:
d, h and d should be in uppercase. Fix the hyperlink (see the example of DKE and check this before submitting for review)